static/index.html

```diff
@@ -333,7 +353,16 @@ <h3>Last error</h3>
           initializeClient: function () {
             var _self = this;
 
-            const _init = function (auth0) {
+            var clientOptions = {
+              domain: _self.domain,
+              client_id: _self.clientId,
+              cacheLocation: _self.useLocalStorage ? 'localstorage' : 'memory',
+              useRefreshTokens: _self.useRefreshTokens,
+              audience: _self.audience,
+              useCookiesForTransactions: _self.useCookiesForTransactions
+            };
```

Comment:
This setup was being duplicated for initializing the client using `createAuth0Client` and from the constructor, so I extracted them out.